src/app/widgets/widget-numeric/widget-numeric.component.ts

```diff
@@ -320,7 +320,7 @@ export class WidgetNumericComponent extends BaseWidgetComponent implements OnIni
 
     let valueText: string = '';
     const maxTextWidth = Math.floor(this.canvasMM.nativeElement.width * 0.45);
-    const maxTextHeight = Math.floor(this.canvasMM.nativeElement.height * 0.075);
+    const maxTextHeight = Math.floor(this.canvasMM.nativeElement.height * 0.13);
```

Comment:
It starts big, but goes small again on first data refresh. This is probably not the right place, or the max size looping bellow is not taking into account.

Reply:
I could have approved the other changes. Try to create PRs that are targeted to one area or feature set. Next time ;)